libs/common-hooks/src/ThemeProvider.tsx

```diff
@@ -0,0 +1,15 @@
+'use client'
+
+import { type PropsWithChildren } from 'react'
+
+import { ThemeProvider as StyledThemeProvider } from 'styled-components/macro'
+
+import type { DefaultTheme } from 'styled-components/macro'
+
+interface ThemeProviderProps extends PropsWithChildren {
+  theme: DefaultTheme
+}
+
+export function ThemeProvider({ children, theme }: ThemeProviderProps) {
```

Comment:
Is this provider in use somewhere?

Reply:
This was from when we I trying to share theme code between apps. But since each app has its own theme setup (with shared types in libs/types), I removed it to clean things up.